src/frontend/src/test-e2e/views.ts

```diff
@@ -126,7 +126,37 @@ export class RecoveryMethodSelectorView extends View {
   }
 
   async getSeedPhrase(): Promise<string> {
-    return await this.browser.$("#seedPhrase").getText();
+
+    // This tries to read the recovery phrase by first copying it to the clipboard.
+
+    await this.copySeedPhrase();
+
+    // Our CSP policy prevents us from directly reading the clipboard.
```

Comment:
For what is worth I don't see this as a mock but sponteneously finds it better than reading directly the clipboard - i.e. in addition to the copy you test also that the OS will paste it correctly

Reply:
yep, 100% agreed that it's better to emulate what the user is doing. However there's no actual user, that's why I'm saying we "mock" the user input :) Happy to reword if necessary!

Reply:
all good